CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.13.0 / 2018-01-12
+
+* [CHANGE] Switch cmd/alertmanager to kingpin (#974)
+* [CHANGE] [amtool] Switch amtool to kingpin (#976)
+* [CHANGE] [amtool] silence query: --expired flag only shows expired silences (#1190)
+* [CHANGE] Return config reload result from reload endpoint (#1180)
+* [CHANGE] Template secret keys for pagerduty notifier (#1182)
```

Comment:
Service keys, and this is a bugfix for a regression. They're templated because they're routing keys, in spite of being secret keys.